tests/drivers/mm/sys_mm_drv_bank/testcase.yaml

```diff
@@ -0,0 +1,3 @@
+tests:
+  drivers.mm.sys_mm_drv_bank:
```

Comment:
Seems like we should have some filter so we only build / run this on platforms that have drivers/mm.

Reply:
with arm-clang we get errors of the form:
```
Error: L6218E: Undefined symbol sys_mm_drv_page_phys_get (referred from mm_drv_common.o).
Error: L6218E: Undefined symbol sys_mm_drv_map_page (referred from mm_drv_common.o).
Error: L6218E: Undefined symbol sys_mm_drv_unmap_page (referred from mm_drv_common.o).
Error: L6218E: Undefined symbol sys_mm_drv_page_flag_get (referred from mm_drv_common.o).
Error: L6218E: Undefined symbol sys_mm_drv_update_page_flags (referred from mm_drv_common.o).
```